analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -146,6 +148,17 @@ public Builder endpoint(String endpoint) {
       return this;
     }
 
+    /**
+     * Set the user agent for this client
```

Comment:
Nit: should have a full stop https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment as if it was a complete sentence.

Could also be in one line:

```
/** Sets a user agent for HTTP requests. */
```